config/jobs/kubernetes/wg-k8s-infra/trusted/sig-security-trusted.yaml

```diff
@@ -49,6 +49,27 @@ periodics:
         else
           echo "Scan completed"
         fi
+
+        # snyk scan on container images
+        echo "Fetch the list of k8s images"
+        curl -L https://sbom.k8s.io/v1.22.1/release | grep 'PackageName: k8s.gcr.io/' | awk '{print $2}' > images
```

Comment:
One thing we might need to explore is how to run a scan always on the latest kubernetes release. Right now 1.22.1 is hardcoded but we might need to find out a way to dynamically inject that. I do not have a good answer for this right now but will try to dig in more

Reply:
yes make sense @PushkarJ . let me dig more into this as well

Reply:
Updated the curl command to run on the latest release